### PR TITLE
Fix documentation about buildTarget

### LINKED ISSUE #2253: Improve docs about launch configuration
**Describe the bug**

We should better explain what is a buildTarget and when it should be specified. We can start with a minimal example and then go into details when anything additional might be needed.

**Search terms**

launch documentation

### PR DESCRIPTION
**Pull request by tgodzik**
Follow up from https://github.com/scalameta/metals/pull/2254 after my breaking push :sweat: 

fix #2253

Thanks @bjornregnell !

### CODE REVIEW

File: docs/editors/vscode.md [hunk 1]
```diff
@@ -177,33 +177,41 @@ guide them. In the end users should end up with something like this:
...
+You can also add an optional build target name, which is needed in case there
+are more than one class with the same name or when launching a class from
+outside the project. Inside `"configurations":` add the key `buildTarget` with
+your target name, e.g. `root`:
+
+```json
+      "buildTarget": "root"
+```
+
+Build target name corresponds to your project name. For example in sbt for
```
Comment (ckipp01) [inline docs/editors/vscode.md:210]:
```suggestion
The build target name corresponds to your project name. For example in sbt for
```

File: docs/editors/vscode.md [hunk 2]
```diff
@@ -177,33 +177,41 @@ guide them. In the end users should end up with something like this:
...
+`lazy val interfaces = project` the name of the build target will be
+`interfaces` for sources and `interfaces-test` for tests. To make sure what are
+the target names please run the command `Metals: Run Doctor`.
```
Comment (ckipp01) [inline docs/editors/vscode.md:213]:
```suggestion
`interfaces` for sources and `interfaces-test` for tests. To make sure you have the correct target names please run the command `Metals: Run Doctor`.
```

**Review by ckipp01 — APPROVED** [review]
Just a couple nits, but other than that, LGTM! Thanks @bjornregnell

File: docs/editors/vscode.md (hunk 2 above)
Comment (tgodzik) [reply docs/editors/vscode.md]: Fixed!

File: docs/editors/vscode.md (hunk 1 above)
Comment (tgodzik) [reply docs/editors/vscode.md]: Fixed

**Pull request merged**